### ISSUE TITLE
Icon and App Notification Issue in POCO F1

### ISSUE THREAD

**Issue by isaumya** [amirzaidi/Launcher3#582]
Hi,
Today I received POCO F1 by Xiaomi device and the first thing I did was installing this launcher in the phone. But After installing the latest version I found a couple of issues.
1. When I go to the launcher settings and click on app notification where it takes me to a different page to enable notification access to the launcher. Every time I try to enable it, the Settings app got crashed. I've already reported the debug details to Xiaomi.
2. The icons always show in square more, even if I go in the launcher settings and select circle as the option. Rest thing was working fine. So, I had to uninstall it and use the default phone launcher.

Also, the launcher is no longer in the Google Play Store. Not sure why.

**Comment by nyancrimew** [comment]
1. Click on the text in the list instead of the toggle and it should work, this is a miui bug
2. This is a miui bug aswell, only square icon shape works there because they broke the standard apis.

**Comment by isaumya** [comment]
@deletescape Thanks for letting me know. Nay idea when miui is going to fix that standard API?

**Comment by amirzaidi** [comment]
1) MIUI bug
2) MIUI bug, might be able to figure out a workaround later

Launcher was suspended because of the automatic installation of the bridge. I'm currently awaiting a response from Google's support.

**Comment by isaumya** [comment]
@amirzaidi has anyone reported this but to MIUI team?

**Comment by vogtmh** [comment]
I've sent them a report when the app crashed. Thanks for info, FINALLY I can use the launcher properly :-) 

For the icons I can recommend the "Pixel Icon Pack" - looks quite pretty in combination with a dark theme and the Rootless Pixel Launcher.

**Comment by isaumya** [comment]
@vogtmh Did you able to use round icons and notification dots?

**Comment by vogtmh** [comment]
Yes, sure. The icon packs provides round icons and have round notification dots as well.

**Comment by isaumya** [comment]
@vogtmh No, I mean trying to enable the notification dots from the settings didn't your settings app got crashed?

**Comment by vogtmh** [comment]
Just click on the text instead of the slider and everything should be fine :-)

**Comment by isaumya** [comment]
@vogtmh are you talking about this icon pack: https://play.google.com/store/apps/details?id=com.themezilla.pixelui ?

**Comment by vogtmh** [comment]
exactly :-)
